### PR TITLE
Data release 2020_10_06_01_01_44

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: golden_response/staging/query/weather2.json [hunk 1]
```diff
@@ -5687,7 +5687,7 @@
           "value": "1943-01"
         },
         {
-          "value": "11.055612903225805"
+          "value": "11.055612903225809"
```
Comment (pradh) [inline golden_response/staging/query/weather2.json:5690]: This is likely some upstream change in the pipeline we consume wether from that roberts owns.

File: golden_response/staging/get_stat_all/result.json [hunk 2]
```diff
@@ -1,765 +1,765 @@
 {
-  "placeData": {
-    "country/USA": {
-      "statVarData": {
-        "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita": {
-          "sourceSeries": [
+  "placeData":  {
```
Comment (pradh) [inline golden_response/staging/get_stat_all/result.json:2]: any idea why this keeps happening?

**Review by pradh — APPROVED** [review]
To confirm, the neighboring places is not part of any test?

**Review by tjann — APPROVED** [review]

**Comment by pradh** [comment]
> To confirm, the neighboring places is not part of any test?

Oh this is the un-viewable large diff on landing page json?

File: golden_response/staging/get_stat_all/result.json (hunk 2 above)
Comment (beets) [reply golden_response/staging/get_stat_all/result.json:2]: not sure. it seems to be get_stat_* related, so we could start there.

**Comment by beets** [comment]
> > To confirm, the neighboring places is not part of any test?
> 
> Oh this is the un-viewable large diff on landing page json?

Yea, and I confirmed that by scrolling to the very bottom where we have just the list of nearby places (and no changes to other related place list)

**Comment by shifucun** [comment]
The extra space from protojson Marshal is still mysterious.

**Comment by beets** [comment]
thanks all!

**Pull request merged**